tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/EmailSendMethod.scala

```diff
@@ -203,7 +206,10 @@ class EmailSendMethod @Inject()(emailSetSerializer: EmailSetSerializer,
         mailImpl.setMessageNoCopy(message)
         mailImpl
       })
-      _ <- SMono(queue.enqueueReactive(mail)).`then`(SMono.just(submissionId))
+      _ <- SMono(queue.enqueueReactive(mail))
+        .`then`(SMono.fromCallable(() => LifecycleUtil.dispose(mail))
+          .subscribeOn(Schedulers.elastic()))
+        .`then`(SMono.just(submissionId))
```

Comment:
While we are at it, LifecycleUtil::dispose is never called on this email. Can we fix that?

Reply:
I don't understand what you mean. 
I just checked, LifecycleUtil.dispose is still invoked when enqueueing